src/DataViews/SamplingReportDataView.cpp

```diff
@@ -53,8 +53,8 @@ const std::vector<DataView::Column>& SamplingReportDataView::GetColumns() {
     columns.resize(kNumColumns);
     columns[kColumnSelected] = {"Hooked", .0f, SortingOrder::kDescending};
     columns[kColumnFunctionName] = {"Name", .4f, SortingOrder::kAscending};
-    columns[kColumnInclusive] = {"Inclusive", .0f, SortingOrder::kDescending};
-    columns[kColumnExclusive] = {"Exclusive", .0f, SortingOrder::kDescending};
+    columns[kColumnInclusive] = {"Inclusive, %", .0f, SortingOrder::kDescending};
+    columns[kColumnExclusive] = {"Exclusive, %", .0f, SortingOrder::kDescending};
```

Comment:
What do you think about removing the comma? I feel like it would be cleaner, but I'm not sure.

Reply:
I think, a shorthand for units should be separated (with a comma or sometimes with brackets). We could write "Inclusive percentage". Like "Length in meters", but "Length, m".

Reply:
I agree with that, it was about the look. Maybe the parentheses look better. In the end all options are fine.